python-regression/tests/features/steps/api_test_steps.py

```diff
@@ -228,3 +232,63 @@ def check_node_sync(step, node, milestone):
         "Latest Solid Milestone {} on {} is not the expected {}".format(latestSolidMilestone, node, milestone)
 
 
+@step(r'A transaction is issued on "([^"]+)" with:')
+def issue_a_transaction(step, node):
+    """
+    Issues a single transaction
+
+    :param node: The identifier for the node (ie nodeA)
+    """
+    world.config['nodeId'] = node
+    api = api_utils.prepare_api_call(node)
+
+    transaction = transactions.evaluate_and_send(api, "", step.hashes)
+    transaction_hash = Transaction.from_tryte_string(transaction.get('trytes')[0]).hash
+    world.responses['transaction_hash'] = transaction_hash
+
+
+@step(r'"(\d+)" milestones are issued on "([^"]+)"')
+def issue_multiple_milestones(step, num_milestones, node):
+    """
+    Issues num_milestones milestones
+
+    :param num_milestones: The number of milestones to issue
+    :param node: The identifier for the node (ie nodeA)
+    """
+    address = static_vals.TEST_BLOWBALL_COO
+    api = api_utils.prepare_api_call(node)
+    seed = transactions.check_for_seed(step.hashes)
+
+    #Latest milestone is 8412
+    for iteration in range(int(num_milestones)):
+        api = api_utils.prepare_api_call(node, seed=seed)
+        logger.info('Issuing milestone {}'.format(iteration + 1))
+        milestone = milestones.issue_milestone(address, api, iteration + 8413)
+
+        milestone_hash = Transaction.from_tryte_string(milestone['trytes'][0]).hash
+        milestone_hash2 = Transaction.from_tryte_string(milestone['trytes'][1]).hash
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'milestone_hash2' is assigned to but never used](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4517274)